addons/mail/static/src/new/utils.js

```diff
@@ -56,19 +48,36 @@ export function onExternalClick(refName, cb) {
 
 export function useAutoScroll(refName) {
     const ref = useRef(refName);
-    let isScrolled = false;
-    onMounted(() => {
-        ref.el.scrollTop = ref.el.scrollHeight;
-    });
-    onWillPatch(() => {
-        const el = ref.el;
+    let el = null;
+    let isScrolled = true;
+    let observer = null;
+    if ("ResizeObserver" in window) {
+        observer = new ResizeObserver(applyScroll);
+    }
```

Comment:
ResizeObserver is supported in all evergreen browser, so I see no reason to check whether it's in `window`.

Also considering how neat `ResizeObserver` is, and the slight cumbersome of API to define an observer and manually `.observe()`/`.unobserve()`, I feel it could be worth defining a hook `useResizeObserver(refName, cb)`.

Just another random idea 🤓

Reply:
wasn't sure about this, but sure, I removed the check. 
for now, I don't feel like we need to define a wrapper, it's just 1 line anyway. maybe later, who knows